docker/phpunit/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM alpine
+
+WORKDIR /tmp
+
+RUN apk --no-cache add \
+	bash \
+	ca-certificates \
+	git \
+	curl \
+	mariadb-client \
+	ncurses \
+	nodejs \
+	npm \
+	subversion \
+	unzip \
+	wget \
+	php7 \
+	php7-bcmath \
+	php7-ctype \
+	php7-curl \
+	php7-dom \
+	php7-exif \
+	php7-json \
+	php7-mcrypt \
+	php7-opcache \
+	php7-openssl \
+	php7-mbstring \
+	php7-mysqli \
+	php7-pcntl \
+	php7-pdo \
+	php7-pdo_pgsql \
+	php7-pdo_sqlite \
+	php7-pdo_mysql \
+	php7-phar \
+	php7-session \
+	php7-soap \
+	php7-tokenizer \
+	php7-xdebug \
+	php7-xml \
+	php7-xmlreader \
+	php7-zip \
+	php7-zlib
```

Comment:
Love it :D 

I like the choice of Alpine and 👏 for figuring out the extensive list of needed dependencies. 🤯